### PR TITLE
fix issue by which pytorch_jni is not bundled in libtorch

### PR DESCRIPTION
**Pull request by erip**
Fixes issue with pytorch_jni.dll not being installed correctly in libtorch.

### CODE REVIEW

**Comment by erip** [comment]
cc @peterjc123 WDYT? We still need upstream changes to land, but it seems like this fixes the immediate issue. It might be a good opportunity to fix #46464, too.

File: android/pytorch_android/CMakeLists.txt
Comment (peterjc123) [inline android/pytorch_android/CMakeLists.txt:168]: One thing you are missing is that you forget to install the PDB fles on debug builds. See https://github.com/pytorch/FBGEMM/blob/master/CMakeLists.txt#L253-L255

**Comment by erip** [comment]
cc @peterjc123 do you think this looks OK to merge once fbjni committers make a move?

**Comment by peterjc123** [comment]
@peterjc123 Yeah. Will ask for someone to land this PR after the one in fbjni is merged.

**Comment by erip** [comment]
Awesome, thanks! And if you know anyone who can help fbjni move, that would be awesome. It seems like fbjni doesn't really have dedicated maintainers.

**Comment by erip** [comment]
@ezyang do you know anyone who could review/merge https://github.com/facebookincubator/fbjni/pull/41? It seems stuck and it's a pretty small PR so once that's merged, I think this is good to go.

**Comment by ezyang** [comment]
I think dreiss will review the fbjni change after dev day, so I think he should be able to handle this one too. @dreiss let me know if you want someone else to, e.g., handle this PR.

**Comment by erip** [comment]
Upstream changes finally landed, so I think this is good to land when CI is green. This has gotten review from @peterjc123. cc @ezyang

**Review by ezyang — APPROVED** [review]

**Comment by erip** [comment]
Woohoo. Looks good! 🎊

**Pull request closed without merging**
